test/governance/extensions/GovernorTimelockAccess.test.js

```diff
@@ -112,6 +114,81 @@ contract('GovernorTimelockAccess', function (accounts) {
         expect(await this.mock.accessManager()).to.be.equal(this.manager.address);
       });
 
+      it('sets base delay (seconds)', async function () {
+        const baseDelay = time.duration.hours(10);
+
+        // Only through governance
+        await expectRevertCustomError(
+          this.mock.setBaseDelaySeconds(baseDelay, { from: voter1 }),
+          'GovernorOnlyExecutor',
+          [voter1],
+        );
+
+        await impersonate(this.mock.address);
+        const { receipt } = await this.mock.setBaseDelaySeconds(baseDelay, { from: this.mock.address });
+        expectEvent(receipt, 'BaseDelaySet', {
+          oldBaseDelaySeconds: '0',
+          newBaseDelaySeconds: baseDelay,
+        });
```

Comment:
IMO it would be better to test that using an actual governance call then using an impersonate.

In prod, you won't be able to impersonate, so we should confirm that governance calls are able to do it. Also, there is way more moving pieces in a governance call (like potentially setting the governanceCall queue) that we want to make sure don't get in the way.

Reply:
This comment applies to basically everywhere "impersonate" is used.

Reply:
Removed all impersonate usage